unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java

```diff
@@ -144,14 +139,22 @@ private static BufferedReader getInputReader(String inputFile) throws IOExceptio
      * @throws IOException
      */
     public static int testInvariants(String inputFile, boolean doRange) throws IOException {
+        if (inputFile == null) {
+            inputFile = DEFAULT_FILE;
+        }
```

Comment:
optional:
Here you could set an extra variable with "" or "-security" so that you need not rediscover later which variant it is.

Or, even better,
- add a `variant` parameter to the testInvariants() signature
- change the call sites to pass it in
- consider having a non-empty suffix for the default tests as well